### ISSUE TITLE
[Rhein and Bremen] CPDLC Freetext

### ISSUE THREAD

**Issue by lucianoasmus** [VATGER-Nav/EDWW-Package-East#137]
Bring back some of the old Freetext mesasges, I miss especially the first one:

```
FREETEXT:R:YOU ARE LEAVING CPDLC COVERAGE REVERT TO VOICE /Bremen Pack only
FREETEXT:NE:REPORT REQUESTED LEVEL
FREETEXT:R:CONTACT ME BY RADIO I HAVE BEEN TRYING TO CALL YOU
```

**Comment by hannesaltmann** [comment]
> Bring back some of the old Freetext mesasges, I miss especially the first one:
> 
> ```
> FREETEXT:R:YOU ARE LEAVING CPDLC COVERAGE REVERT TO VOICE /Bremen Pack only
> FREETEXT:NE:REPORT REQUESTED LEVEL
> FREETEXT:R:CONTACT ME BY RADIO I HAVE BEEN TRYING TO CALL YOU
> ```

The first one is not required anymore as the logoff text has changed.

**Comment by hannesaltmann** [comment]
@ManuelPick könntest du die letzten beiden Free Texts noch hinzufügen?

**Comment by hannesaltmann** [comment]
Fixed with #138
